src/core/constraint_template.jl

```diff
@@ -439,11 +439,12 @@ function constraint_voltage_angle_difference(pm::GenericPowerModel, n::Int, i::I
     branch = ref(pm, n, :branch, i)
     f_bus = branch["f_bus"]
     t_bus = branch["t_bus"]
+    arc_from = (i, f_bus, t_bus)
     pair = (f_bus, t_bus)
     buspair = ref(pm, n, :buspairs, pair)
 
     if buspair["branch"] == i
```

Comment:
This might also be the issue of inconsistent on case 24.  You might be to post this constraint for all branches in the BFM models.

Reply:
In the end, this was not the problem, just a typo in the DistFlow equation for `constraint_voltage_angle_difference`